trading_calendars/exchange_calendar_xasx.py

```diff
@@ -45,7 +46,15 @@
     observance=weekend_to_monday,
 )
 
-AnzacDay = anzac_day(observance=weekend_to_monday)
+# Anzac Day was observed on Monday when it fell on a Sunday in
+# 2010 but that does not appear to have been the case previously.
+# We'll assume that this will be the behavior from now on.
+AnzacDayNonMondayized = anzac_day(end_date='2010')
+AnzacDay = anzac_day(observance=sunday_to_monday, start_date='2010')
+
+# When Easter Monday and Anzac Day coincided in 2011, Easter Tuesday
+# was also observed as a public holiday.
+EasterTuesday2011AdHoc = Timestamp('2011-04-26', tz='UTC')
```

Comment:
It happens about once a century or so, and next happens in 2095. I figured that it would be difficult to tell whether this will be handled the same well at that point, so left it as a single ad hoc.

Reply:
in that case, we can cross that bridge when we get to it :)

Reply:
I'll include this in the comment in case others have the same question